.github/workflows/ci.yml

```
@@ -109,7 +109,8 @@ jobs:
   bazel:
     name: Bazel on ${{matrix.os == 'ubuntu' && 'Linux' || matrix.os == 'macos' && 'macOS' || matrix.os == 'windows' && 'Windows' || '???'}}
     runs-on: ${{matrix.os}}-latest
-    if: github.event_name != 'pull_request'
+    # DO NOT MERGE
+    # if: github.event_name != 'pull_request'
```

This should be reverted before merging.
```suggestion
    if: github.event_name != 'pull_request'
```

---

Dropped this change.